test/integration/resultsPage.js

```diff
@@ -40,10 +40,48 @@ describe('Results page', () => {
 describe('Results page error handling', () => {
   describe('when search is not included', () => {
     it('should return a descriptive error messages', (done) => {
+      const search = null;
+      const errorMessage = messages.emptySearchMessage();
```

Comment:
On a second thought there is a wider discussion here, would it be for integration tests, for unit tests, for both (we are using it for both at the moment)? Can we postpone this request until we have a clear guideline in testing please?